### ISSUE TITLE
[Browser Bookmarks] - maxObjectCount exceeded when selecting browser

### ISSUE THREAD

**Issue by makaidos42** [raycast/extensions#6327]
### Extension

https://raycast.com/raycast/browser-bookmarks

#### Error:
```
Error: maxObjectCount exceeded
    at Ie.parseBuffer (/Users/kane.vandenbiggelaar/.config/raycast/extensions/254a620a-7fba-450b-bc17-c2e1ce6809d8/index.js:34:60403)
    at a9 (/Users/kane.vandenbiggelaar/.config/raycast/extensions/254a620a-7fba-450b-bc17-c2e1ce6809d8/index.js:36:24467)
    at /Users/kane.vandenbiggelaar/.config/raycast/extensions/254a620a-7fba-450b-bc17-c2e1ce6809d8/index.js:36:25009
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3)
```

### Steps To Reproduce

1. Download the "Search Browser Bookmarks" updated extension
2. Select "Safari" as the browser
3. Receive the "failed to fetch latest data: maxObjectCount exceeded" error

Please note that I do have over 3000 bookmarks in Safari...
This did work in the previous (now deprecated) version of this extension.

### Expected Behaviour

Like the previous version of this extension, I would expect that the updated extension could read the bookmarks from my browser.

**Comment by thomaslombart** [comment]
Thanks for flagging this issue. I’ve added it to our team’s backlog, but it may take some time before we can implement it. 

In the meantime, feel free to submit a pull request if you’d like to tackle this issue. I’d be happy to review it! You can contact me here or in the [Slack community](http://raycastcommunity.slack.com/) if you need further help.

**Comment by thomaslombart** [comment]
Reverting it as the fix in #6585 introduced another error.
